echopype/convert/parse_azfp.py

```diff
@@ -243,6 +263,10 @@ def _test_valid_params(params):
                         self.unpacked_data["tilt_y"].append(
                             self._compute_tilt(ping_num, "Y", tilt_y_is_valid)
                         )
+                        # Compute pressure from unpacked_data[ii]['ancillary'][3]
```

Comment:
A cosmetic change: move the `self.unpacked_data["pressure"]` statement to be right after `self.unpacked_data["temperature"]` statement. Temperature and pressure come from external sensors that are sometimes not present (especially pressure). In that sense, I think it's a bit more logical to have those two variables be together. Also, here the pressure statement is in between statements involving tilt, which should remain as a continuous block.

Reply:
Updated!